collections/_article/dev-snapshot-godot-3-6-beta-1.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Dev snapshot: Godot 3.6 beta 1"
+excerpt: ""
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/dev-snapshot-godot-3-6-beta-1.jpg
+image_caption_title: Your Only Move Is HUSTLE
+image_caption_description: A game by Ivy Sly
+date: 2023-04-13 15:00:00
+---
+
+Our biggest milestone in 4 years, Godot 4.0, has just been [released a month ago](/article/godot-4-0-sets-sail/) (and already got [a couple](/article/maintenance-release-godot-4-0-1/) maintenance [fixes](/article/maintenance-release-godot-4-0-2/)). The reception has been great, and we're already hard at work on the 4.1 milestone with a bunch of improvements.
+
+But we're not forgetting about users with Godot 3 projects, which they plan to maintain for months or years to come. The previous stable branch, Godot 3.5, is still receiving [maintenance updates](/article/maintenance-release-godot-3-5-2/), and last but not least, we're working on Godot 3.6 as a feature update to the 3.x branch.
+
+Development on Godot 3.6 is slower paced, as most contributors focus on the 4.x branch, which is where the future of Godot lies. But little by little, improvements and bugfixes are being backported to the 3.x branch and we've accumulated enough of those now to propose a **3.6 beta 1** to start the testing phase leading to the stable release.
```

Comment:
```suggestion
Our biggest milestone in 4 years, Godot 4.0, has just been [released a month ago](/article/godot-4-0-sets-sail/). The reception has been great, and we're already hard at work on the 4.1 milestone with a bunch of improvements. But that doesn't mean we're forgetting Godot 3 users and their projects, which they plan to maintain for months or years to come.

The current stable version, Godot 3.5, has just received a new [maintenance update](/article/maintenance-release-godot-3-5-2/), addressing several reported issues. At the same time we've been working on the next feature update to the 3.x branch — Godot 3.6.

The development is slower paced, as most contributors focus on the 4.x branch, which is where the future of Godot lies. But little by little, improvements and bugfixes are being backported and we've accumulated enough of those now to propose a **3.6 beta 1**, to start the testing phase leading to the stable release.
```

Reply:
Technically the current stable version is 4.0.2 :)

3.5.2 is the previous/old stable branch.